src/ReactNativeImagePicker.re

```diff
@@ -70,24 +70,24 @@ module ImagePicker = {
   };
 
   type response = {
-    .
-    "didCancel": bool,
-    "error": option(string),
-    "customButton": string,
-    "data": string,
-    "uri": string,
-    "origURL": option(string),
-    "isVertical": bool,
-    "width": int,
-    "height": int,
-    "fileSize": int,
-    "type": option(string),
-    "fileName": option(string),
-    "path": option(string),
-    "latitude": option(float),
-    "longitude": option(float),
-    "timestamp": int,
-    "originalRotation": float,
+    didCancel: bool,
+    error: option(string),
+    customButton: string,
+    data: string,
```

Comment:
I think the type of `data` should be `option`. The [doc](https://github.com/react-native-community/react-native-image-picker/blob/master/docs/Reference.md#options) says we can pass `noData` option to disable generating data. Also, `data` is not generated for video.

Reply:
@a-c-sreedhar-reddy care to comment or update?